geth/proxy/transactions/sendtransaction.go

```diff
@@ -0,0 +1,299 @@
+package transactions
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/les/status"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// contains sets of possible defines the name for a giving transaction.
+const (
+	SendTransactionName = "eth_sendTransaction"
+)
+
+// ExecuteRemoteSendTransaction defines a funct
+func ExecuteRemoteSendTransaction(manager common.RPCNodeManager, req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
+	resp, _ := call.Otto.Object(`({"jsonrpc":"2.0"})`)
+	resp.Set("id", req.ID)
+
+	config, err := manager.NodeConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	selectedAcct, err := manager.Account().SelectedAccount()
+	if err != nil {
+		return nil, err
+	}
+
+	nonce := uint64(req.ID)
```

Comment:
As I understand, this `req.ID` refers to JSON-RPC call id and it shouldn't be the nonce.
I think we should only query `GerTransactionCount` in a single place and only when we're sending it to an upstream node.